main.go

```diff
@@ -134,6 +147,86 @@ func main() {
 		os.Exit(1)
```

Comment:
I'm wondering if the reason the audit test is failing is that there is a deadlock here when the webhook is not enabled? That or a fight between the two admission controllers that causes one to never report success? We should figure out that root cause.

After that, we should may sure we are gating cert generation both on whether we have disabled cert rotation and whether the webhook is enabled (`operations["webook"]` evaluates to `true`)

Reply:
I think the reason the audit pod never starts is because the secret is never mounted, so the cert bootstrap check never passes.

Reply:
The cert generation only depends on `disableCertRotation` flag. I noticed, similar to the 'manager' pod, the 'audit' pod may or may not have restarts. The issue is in the `writeFn()` in updating the secret.

Also, the audit test fails even when the audit pod is already running.

Reply:
I'm guessing this comment is no longer relevant?

Reply:
Right, since we enable certRotation only when both `disableCertRotation` and `operation["webhook"]` meet the condition.